test/smoke/interior/interior_suite_test.go

```diff
@@ -0,0 +1,19 @@
+package interior_test
+
+import (
+	"github.com/rh-messaging/qdr-shipshape/pkg/testcommon"
+	"github.com/rh-messaging/shipshape/pkg/framework/log"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	testcommon.Initialize(m)
+}
+
+func TestInterioredge(t *testing.T) {
```

Comment:
One more Renato. Lets change it as it might be helpful when dealing with reporting.

Reply:
Changed